builds/jazz-build-module/sls-app/whitelist-validator-module.groovy

```diff
@@ -226,4 +226,8 @@ def checkAssetType( resourceType ) {
   return status
 }
 
+def getWhitListYml () {
```

Comment:
typo `getWhiteListYml` remove space between name and `()`

Reply:
Done